dmscripts/models/queries.py

```diff
@@ -64,3 +64,10 @@ def add_counts(join, group_by, model, data, directory):
         )['_count'].fillna(0)
 
     return data
+
+
+def expand_json_subfields(field, subfields, data):
```

Comment:
Flatten sounds about right in this instance
https://stackoverflow.com/questions/20957023/flattening-a-binary-tree-to-an-array-in-java